src/Grphp/grpc.stubs.php

```diff
@@ -23,8 +23,7 @@
 const STATUS_UNKNOWN = 2;
 const STATUS_OK = 0;
 
-
-if (!defined(\Grpc\ChannelCredentials::class)) {
+if (!class_exists(\Grpc\ChannelCredentials::class, false)) {
```

Comment:
Nice catch 👏 
We probably need to make this trigger autoloading though as it will return false if the class is available via the autoloader but hasn't been used yet.

Reply:
🙇 Thank you!